application/src/components/views/CityView.scss

```diff
@@ -0,0 +1,20 @@
+#cityhallimg{
+    left: 100px;
+    width: 100px;
+    height: 100px;
+    background: url('../../../public/images/city.png') 100px 100px;
```

Comment:
There has to be a better way around this, we should not rely on relative imports. never